trollflow2/tests/test_launcher.py

```diff
@@ -581,6 +581,32 @@ def wait(job):
         time.sleep(0.11)
 
 
+def test_workers_initialized():
+    """Test that the config loading works when workers are defined."""
+    from tempfile import NamedTemporaryFile
+    import os
+
+    queue = mock.MagicMock()
+
+    with NamedTemporaryFile(mode='w+t', delete=False) as tmp_file:
+        fname = tmp_file.name
+        tmp_file.write(yaml_test_minimal)
+        tmp_file.close()
+
+        try:
+            with mock.patch("trollflow2.launcher.get_dask_client") as gdc:
+                # `get_dask_client()` is called just after config reading, so if we get there loading worked
+                gdc.side_effect = StopIteration
+                try:
+                    process("msg", fname, queue)
+                except StopIteration:
+                    pass
+                else:
+                    raise
```

Comment:
codecov says this is not covered. Is this actually necessary in case the bug comes back?

Reply:
Apparently not. Removed.